### DISCUSSION TITLE
ONN TV Android TV box octo4a

### DISCUSSION THREAD

**Discussion by flyingmonkey35** [feelfreelinux/octo4a#136] [General]
so for Giggles, I picked up a 19 dollar TV box from Walmart,  the ONN TV box. I was Succesful able to Sideload octa4a,  But then I ran into issues.

While the app showed as loaded,  it does not actually display nor can i log into it.  nor does it allow me to launch it.

I believe this is a restriction locking this from only running TV shows.   DAM  this would be a super cheap octoprint box. 

the goal is to make this a simple as possible so i do not want to get into flashing the device or putting custom roms on it.

thoughts.

**Comment by Arsenich14** [comment]
Hello!
Also want to make an android TV box as octoprin(or anover print) server. I've found several decisions for it.
The first one is installing octoprint on tv box, but it needs special Linux firmware (Armbian)
The second is octo4a, which I think easyer, because it installs on Android without "dancing with a tambourine"...
TV box is s905w based Tanix TX 3 Max 2/16Gb.
Soon I'll start testing it, but did anyone already launched it?

**Reply by Arsenich14** [reply]
Yesterday realised these idea. There were problems with startup, first start after installing was good, but next times it crashed... The problem was with webcam server(I don't use it now), after I switched it off in settings everything normalised, printing started. Thannk you for octo4a)
